tol_lab_share/messages/traction/reception_message.py

```diff
@@ -48,6 +48,7 @@ def __init__(self):
         self.supplier_name: str | None = None
         self.taxon_id: str | None = None
         self.template_prep_kit_box_barcode: str | None = None
+        self.retention_instruction: str = "return_to_customer_after_2_years"
```

Comment:
Is this defining the default value for this field?  We don't need to (and shouldn't) do that here.  The schema defines the default value.  What we DO need to do is update our message parser in the `lab-share-lib` repository so that it uses two schemas to decode messages, not just the one like it is currently set up to do.  Then if you send a v1 message to TOL lab share, it will use the v1 and the v2 schema to identify the default value.